homeassistant/components/homewizard/sensor.py

```diff
@@ -22,282 +26,335 @@
         key="voltage_sag_l1_count",
         name="Voltage sags detected L1",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_sag_l1_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="voltage_sag_l2_count",
         name="Voltage sags detected L2",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_sag_l2_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="voltage_sag_l3_count",
         name="Voltage sags detected L3",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_sag_l3_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="voltage_swell_l1_count",
         name="Voltage swells detected L1",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_swell_l1_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="voltage_swell_l2_count",
         name="Voltage swells detected L2",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_swell_l2_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="voltage_swell_l3_count",
         name="Voltage swells detected L3",
         icon="mdi:alert",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.voltage_swell_l3_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="any_power_fail_count",
         name="Power failures detected",
         icon="mdi:transmission-tower-off",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.any_power_fail_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="long_power_fail_count",
         name="Long power failures detected",
         icon="mdi:transmission-tower-off",
         entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda data: data.long_power_fail_count,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="active_power_average_w",
         name="Active average demand",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        value_fn=lambda data: data.active_power_average_w,
     ),
-    SensorEntityDescription(
+    HomeWizardSensorEntityDescription(
         key="montly_power_peak_w",
         name="Peak demand current month",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        value_fn=lambda data: data.montly_power_peak_w,
```

Comment:
This looks like a typo, but I guess that comes from the library or even the provider...
It's existing so can be ignored I guess.

Reply:
It is from upstream indeed, if I fix it here it breaks 😉

Reply:
🤦‍♂️ ... will fix this somehow later on (as it also exists in the API itself)